public/app/features/dashboard/components/PanelEditor/PanelOptionsTab.tsx

```diff
@@ -168,7 +168,14 @@ export const PanelOptionsTab: FC<Props> = ({
   );
 
   if (config.featureToggles.panelLibrary) {
-    elements.push(<PanelLibraryOptionsGroup panel={panel} dashboard={dashboard} key="Panel Library" />);
+    elements.push(
+      <PanelLibraryOptionsGroup
+        panel={panel}
+        dashboard={dashboard}
+        onChange={() => onPanelConfigChange('isEditing', true)}
```

Comment:
Nit: maybe try not to use anonymous arrow functions that get recreated on each re-render.